src/Header/SetCookie.php

```diff
@@ -378,7 +378,7 @@ public function getMaxAge()
      */
     public function setExpires($expires)
     {
-        if ($expires === null) {
+        if ($expires === null || $expires === '') {
```

Comment:
@Maks3w I know, but a cookie is just a header. Also the class name confirms this: `Zend\Http\`**`Header`**`\SetCookie`.